src/components/renderers/renderer-service.spec.ts

```diff
@@ -0,0 +1,30 @@
+import "angular"
+import "angular-mocks"
+
+import IRootScopeService = angular.IRootScopeService;
+import ICompileService = angular.ICompileService;
+
+describe("Renderer service", () => {
+
+
+    beforeEach(angular.mock.module('jsonforms.form'));
+
+    xit("should support remote references",
```

Comment:
This test will fail, I should've probably pushed it in a separate commit. `xit` causes the test to be ignored.